app/models/customer.rb

```diff
@@ -2,4 +2,16 @@ class Customer < ApplicationRecord
   has_many :invoices, dependent: :destroy
   has_many :transactions, through: :invoices
   has_many :items, through: :invoices
+
+  def self.top_five_cust
+    joins(:transactions).select("customers.*, count(transactions.id) as transaction_count").where("result = 1").group(:id).order("transaction_count desc").limit(5)
+  end
+
+  def success_count
+    transactions.success.count
+  end
+
+  def full_name
```

Comment:
I like having the full_name helper method to clean things up!

Reply:
And yes I was thinking the same thing, helper method to unify `first_name` `last_name`